.github/workflows/test.yml

```diff
@@ -0,0 +1,78 @@
+name: Tests
```

Comment:
Could we consider a more descriptive name than `test.yml`?

Reply:
It sort of hits the nail on the head  though, it is the test workflow, and its `.github/workflows/test.yml`.

Reply:
Its also pretty standard way of naming workflows: https://github.com/numpy/numpy/tree/main/.github/workflows

Reply:
Ok, fair enough.